### ISSUE TITLE
doesnt work anymore after upgrade to vscode 1.17.2

### ISSUE THREAD

**Issue by sennierer** [flathub/com.visualstudio.code.tool.podman#22]
after upgrading to the latest vscode version I dont get the podman-remote command in vscode anymore and vscode is complaining that I need to install docker.

**Comment by gasinvein** [comment]
What was the previous version you were using? Make sure you have `com.visualstudio.code.tool.podman//21.08` (note the `21.08` branch) installed.

**Comment by sennierer** [comment]
yes, thats the one I have installed (even reinstalled it to make sure). I also re-ran 'flatpak override --user --filesystem=xdg-run/podman:ro com.visualstudio.code'

**Comment by gasinvein** [comment]
Oh, you're using the proprietary VSCode build, then? This repo is for Code-OSS.

**Comment by sennierer** [comment]
:) interesting, worked before the upgrade

**Comment by gasinvein** [comment]
This doesn't reproduce for me with Code-OSS, so I gonna transfer this issue to the proprietary VSCode flatpak repo, maybe it's maintainers have an idea.
